app/models/provider.rb

```diff
@@ -64,4 +65,10 @@ def address_info
       .attributes_before_type_cast
       .slice('address1', 'address2', 'address3', 'address4', 'postcode', 'region_code')
   end
+
+  def update_changed_at(timestamp: Time.now.utc)
+    # We don't want `updated_at` to change, only `changed_at`. Otherwise what
+    # would be the point of separating the two?
```

Comment:
I'm not sure we should pose a question in the comments. How about explaining the separation
"changed_at represents changes to related records as well as provider itself, so we don't want to alter the semantics of updated_at which represents changes to just the provider record"